docs/content/any/project/changelogs/2021-06-09.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Release 2021-06-09
+menuTitle: 2021-06-09
+any: true
+description: >-
+  Changelog for Release 2021-06-09 (oso 0.12.4, sqlalchemy-oso 0.7.1) containing new features,
+  bug fixes, and more.
+---
+
+## `oso` v0.12.4
+
+### Other bugs & improvements
+
+- Fixed bug with dot lookups in the head of a rule ([#933](https://github.com/osohq/oso/pull/933)).
+- Fixed some memory leaks in the go library ([#944](https://github.com/osohq/oso/issues/944)).
+
+## `sqlalchemy_oso` v0.8.0
+
+### Core
+
+The sqlalchemy roles library, previously published as `sqlalchemy-oso-preview` is no longer in preview and included
+as part of the `sqlalchemy-oso` package.
+
+#### Breaking changes
```

Comment:
This is missing the main breaking change that `sqlalchemy_oso.roles` is now the new library, and the old library is deprecated and under a different module path.